packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuButton.deprecated.test.tsx

```diff
@@ -1,49 +0,0 @@
-import * as React from 'react';
-import * as renderer from 'react-test-renderer';
-import { IContextualMenuItem } from '../ContextualMenu.types';
-import { IMenuItemClassNames } from '../ContextualMenu.classNames';
-import { ContextualMenuButton } from './ContextualMenuButton';
-
-describe('ContextualMenuButton', () => {
-  describe('creates a normal button', () => {
-    let menuItem: IContextualMenuItem;
-    let menuClassNames: IMenuItemClassNames;
-
-    beforeEach(() => {
-      menuItem = { key: '123' };
-      menuClassNames = getMenuItemClassNames();
-    });
-
-    it('renders the contextual menu split button correctly', () => {
-      const component = renderer.create(
-        <ContextualMenuButton
-          item={menuItem}
-          classNames={menuClassNames}
-          index={0}
-          focusableElementIndex={0}
-          totalItemCount={1}
```

Comment:
I'll double check, but when I deleted these early on it was because the `*.deprecated.test.tsx` were identical to their `*.test.tsx` file aside from my refactor changing the import paths

![image](https://user-images.githubusercontent.com/706967/44179297-0f384d80-a0ab-11e8-8201-0fcad49e6df2.png)

Will go thru them once more to confirm that's the case for all that were removed.

Reply:
Spoke offline and now understand the purpose of these test files :)